sdks/java/core/src/main/java/org/apache/beam/sdk/util/WindowedValue.java

```diff
@@ -187,13 +182,6 @@ public T getValue() {
     return windowedValues.build();
   }
 
-  /**
-   * Returns the pane of this {@code WindowedValue} in its window.
-   */
-  public PaneInfo getPane() {
-    return pane;
-  }
-
   @Override
   public abstract boolean equals(Object o);
```

Comment:
Good idea. What I've done is made equality part of the spec for any `WindowedValue`. Now the implementations present in the implementations here are just fast paths.